annotations/build.gradle

```diff
@@ -1,6 +1,14 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 apply plugin: 'org.jetbrains.kotlin.jvm'
 apply plugin: 'com.vanniktech.maven.publish'
 
 kotlin {
-  explicitApi = 'strict'
+  explicitApi()
+}
+
+tasks.withType(KotlinCompile).configureEach {
+  kotlinOptions {
+    freeCompilerArgs += ["-Xopt-in=kotlin.RequiresOptIn"]
+  }
```

Comment:
This could be added to the root `build.gradle` file. Or would that be a bad idea?

Reply:
I tend to default to being explicit but don't have strong feelings about it if you'd rather it be in the root

Reply:
Ah, leave it as is if it's easier for you and you didn't make the change yet.

Reply:
Done!